atlassian_jwt_auth/contrib/tests/utils.py

```diff
@@ -1,20 +1,21 @@
 import atlassian_jwt_auth
 
 
-class StaticPublicKeyRetriever(object):
```

Comment:
What value does changing this code add? (I don't see any value except code churn ...).